.github/workflows/prepare-prerelease.yml

```diff
@@ -0,0 +1,64 @@
+on:
+  push:
+    branches: [ master ]
+
+jobs:
+  prepare-release:
+    runs-on: ubuntu-latest
+    if: startsWith(github.event.head_commit.message, '[Release:beta]')
+    steps:
+      - uses: actions/checkout@v3
+        with:
+          token: ${{ secrets.BOT_GITHUB_TOKEN }}
```

Comment:
this should be an input

Reply:
Same as above: this is not a reusable workflow.